AdyenCard/Form/FormCoBadgedCardItem.swift

```diff
@@ -0,0 +1,125 @@
+
+    /// The `accessibilityIdentifier` to be used on the `FormCoBadgedCardItem`
+    internal var identifier: String?
+
+    /// The `accessibilityLabel` to be used on the ``FormCoBadgedCardItem``
+    internal let accessibilityLabel: String
+
+    internal var isHidden: AdyenObservable<Bool> = AdyenObservable(true)
+
+    /// The coBadged card item style.
+    internal var style: FormCoBadgedCardItemStyle
+
+    /// The callback to send the selected brand to the Card Viewcontroller
+    internal var onCardBrandSelection: ((CardBrand) -> Void)?
+
+    /// Brands set after coBadgedCardItems are displayed on the view
+    @AdyenObservable(nil) internal private(set) var cardItemsDisplayed: [CardBrand]?
+
+    /// Initializes the FormCoBadged card item.
+    ///
+    /// - Parameters:
+    ///   - title: The title of the coBadged card selection UI.
+    ///   - subtitle: The subtitle of the coBadged card selection UI.
+    ///   - selectableFormItems: The array of the selectableFormItems.
+    ///   - style: The style of the coBadged card form item.
+    ///   - identifier: The `accessibilityIdentifier` to be used on the `FormCoBadgedCardItem`
+    ///   - accessibilityLabel: An optional custom `accessibilityLabel` to use.
+    internal init(
+        title: String,
+        subtitle: String,
+        selectableFormItems: [SelectableFormItem],
+        style: FormCoBadgedCardItemStyle,
+        identifier: String? = nil,
+        accessibilityLabel: String? = nil
+    ) {
+        self.title = title
+        self.subtitle = subtitle
+        self.style = style
+        self.identifier = identifier
+        self.accessibilityLabel = accessibilityLabel ?? ""
+        self.selectableFormItems = selectableFormItems
+    }
+
+    // MARK: - BuildableFormItem
+
+    internal func build(with builder: FormItemViewBuilder) -> AnyFormItemView {
+        builder.build(with: self)
+    }
+
+    internal func selectableFormItems(
+        from brands: [CardBrand],
+        cardLogos: [FormCardLogosItem.CardTypeLogo],
+        defaultSelectedBrand: CardBrand
+    ) -> [SelectableFormItem] {
+        brands.map { brand in
+            let brandLogoURL = cardLogos.first(where: { $0.type == brand.type })?.url
+
+            let isSelected = brand.type.rawValue == defaultSelectedBrand.type.rawValue ? true : false
+
+            let selectableItem = SelectableFormItem(
+                title: brand.type.name,
+                imageUrl: brandLogoURL,
+                isSelected: isSelected,
+                style: .init(title: style.title),
+                identifier: brand.type.rawValue
+            )
+            selectableItem.selectionHandler = { [weak self] in
+                guard let self else { return }
+                onCardBrandSelection?(brand)
+            }
+            return selectableItem
+        }
+    }
+
+    internal func updateSelectableFormItems(_ brands: [CardBrand], cardLogos: [FormCardLogosItem.CardTypeLogo]) {
```

Comment:
No need to mention selectableFormitems name in explicitly, we can name it to `updateItems(...`
(and `resetItems()` if you prefer a different function to set the array to empty)